packages/live-share-media/src/LiveMediaSessionCoordinator.ts

```diff
@@ -675,4 +671,25 @@ export class LiveMediaSessionCoordinator extends EventEmitter {
                     : 0.0;
         }
     }
+
+    private async onReceivedConnectEvent(
+        evt: ILiveEvent<undefined>,
+        sender: string,
+        local: boolean
+    ): Promise<boolean> {
+        // Immediately send a position update
+        try {
+            const state = this._getPlayerState();
```

Comment:
Can we keep the telemetry event on connected?

Reply:
It was, the events were generated externally, but I added it back as it should still be useful since there isn't specific telemetry around "connect".